scripts/sword_locales_to_json.mjs

```diff
@@ -0,0 +1,262 @@
+      indexedLangs[currentLang.iso6391] = currentLang;
+    }
+    if (currentLang.iso6393) {
+      indexedLangs[currentLang.iso6393] = currentLang;
+    }
+  }
+
+  return indexedLangs;
+}
+
+function addI18nData(code, localeCodes, data = {}) {
+  // Try to get localized name through standard Internationalization API, requires node >= 14
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames/of
+  if (hasFullICU) {
+    for (const localeCode of localeCodes) {
+
+      if (code.length >= 2 && code.length <= 3) {
+        const languagesInLocale = new Intl.DisplayNames(localeCode, { type: 'language', fallback: 'none' });
+        const languageName = languagesInLocale.of(code);
+        if (languageName) {
+          data[localeCode] = toTitleCase(languageName);
+        }
+      } else if (code.length == 4) {
+        const languageScript = (new Intl.DisplayNames(localeCode, { type: 'script', fallback: 'none' })).of(code);
+        if (languageScript) {
+          data[localeCode] = languageScript;
+        }
+      } else {
+        console.log(`Non standard language code "${code}" for Intl API of "${localeCode}"`);
+        break;
+      }
+
+      if (data.regions) {
+        for (const regionCode in data.regions) {
+          const languageRegion = (new Intl.DisplayNames(localeCode, { type: 'region', fallback: 'none' })).of(regionCode);
+          if (languageRegion) {
+            data.regions[regionCode][localeCode] = languageRegion;
+          }
+        }
+      }
+
+    }
+  }
+  return data;
+}
+
+async function getEzraBibleAppLocales(url) {
+  return new Promise((resolve, reject) => {
+    https.get(url, (res) => {
+      const { statusCode } = res;
+
+      let error;
+      // Any 2xx status code signals a successful response but
+      // here we're only checking for 200.
+      if (statusCode !== 200) {
+        error = new Error(`Request ${url} failed.\n` +
+          `Status Code: ${statusCode}`);
+      }
+      if (error) {
+        console.error(error.message);
+        // Consume response data to free up memory
+        res.resume();
+        reject(error);
+        return;
+      }
+
+      res.setEncoding('utf8');
+      let rawData = '';
+      res.on('data', (chunk) => { rawData += chunk; });
+      res.on('end', () => {
+        try {
+          const parsedData = JSON.parse(rawData);
+          console.log(parsedData.available);
+          resolve(parsedData.available);
+        } catch (err) {
+          console.error(err.message);
+          reject(err);
+        }
+      });
+    }).on('error', (err) => {
+      console.error(`Got error: ${err.message}`);
+      reject(err);
+    });
+  });
+}
+
+function toTitleCase(str) {
+  return str.slice(0, 1).toLocaleUpperCase() + str.slice(1);
+}
+
+function saveToJsonFile(data, filename) {
+  const jsonData = JSON.stringify(data);
+  // const jsonData = JSON.stringify(data, null, 2); // If pretty print is needed
```

Comment:
Would this also result in an output with linebreaks? I would prefer the pretty print version actually.

Reply:
Yes, it will be with the line breaks.

Reply:
Ok, then I would definitely suggest to switch to the pretty print version.